src/main/webapp/css/user-page.css

```diff
@@ -2,30 +2,83 @@ body > .page-header {
   margin: 7em 0em 4em 0em;
 }
 
+.content-container {
+  width: 80%;
+  margin: auto;
+  display: flex;
+}
+
+.left-container {
+  flex: 30%;
+  top: 7em;
+  height: 90vh;
+}
+
+#user-profile-image {
+  object-fit: cover;
+  width: 290px;
+  height: 290px;
+}
+
+profile-container {
```

Comment:
This CSS class needs a `.` in front of it to be applied correctly I think

Reply:
Oh yes!